spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/LeakyReLU.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
+
+import com.intel.analytics.bigdl._
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.nn.abstractnn._
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
+
+import scala.reflect.ClassTag
+
+
+@SerialVersionUID( - 1470253389268877486L)
+class LeakyReLU[T: ClassTag](private val alpha: Double = 0.01,
```

Comment:
Why this alpha is private? cc @zhichao-li

Reply:
Pls confirm this with the original author. we can open it if no objections.

Reply:
In the original nn/LeakyReLU, the negval is private so I set the alpha as private.

Reply:
cc @psyyz10 @qiuxin2012 Any comments on this?